include/core/CJsonStateRestoreTraverser.h

```diff
@@ -140,7 +294,12 @@ class CORE_EXPORT CJsonStateRestoreTraverser : public CStateRestoreTraverser {
             E_TokenObjectStart = 9,
             E_TokenObjectEnd = 10,
             E_TokenArrayStart = 11,
-            E_TokenArrayEnd = 12
+            E_TokenArrayEnd = 12,
+            E_TokenKeyPart = 13,
+            E_TokenStringPart = 14,
+            E_TokenComma,
+            E_TokenColon,
+            E_TokenWhiteSpace
```

Comment:
It should be possible to write the rest of the parser in such a way that these extra constants are not needed.